test/unit/LedgerId.js

```diff
@@ -0,0 +1,86 @@
+import LedgerId from "../src/LedgerId.js";
+import * as hex from "../src/encoding/hex.js";
+
+describe("LedgerId", function () {
+    const MAINNET = "mainnet";
+    const TESTNET = "testnet";
+    const PREVIEWNET = "previewnet";
+
+    it('fromString returns LedgerId from strings/hex "mainnet", "testnet", "previewnet"', function () {
+        let ledgerId = LedgerId.fromString("0");
+        expect(ledgerId.ledgerId).to.eql(new Uint8Array([0]));
+        ledgerId = LedgerId.fromString(MAINNET);
+        expect(ledgerId.ledgerId).to.eql(new Uint8Array([0]));
+        ledgerId = LedgerId.fromString(hex.encode(MAINNET));
```

Comment:
Um, this should not work. `hex.encode()` requires `Uint8Array` to be provided not `string`. If this does not error that's unfortunate.